public/app/plugins/panel/singlestat/module.ts

```diff
@@ -182,9 +182,9 @@ class SingleStatCtrl extends MetricsPanelCtrl {
     if (tableData.columns.length === 1) {
       this.panel.tableColumn = tableData.columns[0].text;
     } else {
-      this.panel.tableColumn = _.find(tableData.columns, col => {
+      this.panel.tableColumn = _.find(tableData.columns!, col => {
         return col.type !== 'time';
-      }).text;
+      })!.text;
```

Comment:
can columns and text nullable/undefined? that can't be right

Reply:
`_.find` can return `undefined`, not sure if it ever will in this case though.

Reply:
I added `tableColumn` as an array of MutableColumns and undefined. Would this be a correct approach?